src/Framework/Menus/CustomBoard.cs

```diff
@@ -16,9 +16,10 @@ class CustomBoard : IClickableMenu
     {
         public static readonly Dictionary<string, Quest> todayQuests;
 
-        private readonly ClickableComponent _acceptQuestButton;
+        public ClickableComponent AcceptQuestButton;
         private readonly Texture2D _billboardTexture;
         private readonly Quest _offeredQuest;
+        private readonly string _boardName;
```

Comment:
Why? This field is not referenced, it's only assigned, but never used. So this field is useless and can be removed.

Reply:
It's need to call the constructor of CustomBoard, see issue below.

Reply:
ok